packages/selleo-design-core/src/Testimonials.tsx

```diff
@@ -0,0 +1,73 @@
+import { h } from "preact";
+import { AvatarProps } from "./Avatars";
+
+type TestimonialProps = {
+  regularAvatar?({ hideName, imageSize }: AvatarProps): h.JSX.Element;
+};
+
+const avatar = (
+  <img
+    src="https://ui-avatars.com/api/?name=John+Doe"
+    alt="avatar"
+    class="rounded-full w-[100%]"
+  />
+);
+
+export function BigTestimonial({ regularAvatar }: TestimonialProps) {
```

Comment:
please create a one testimionial component with `variant` prop

Reply:
that will be regular / big / vertical ( i know it's not sematically correct since we mixing sizes and variant but lets simplify it right now. like that)

Reply:
Okay, done